### ISSUE TITLE
Kalyn-Ben Meeting 2020-02-24

### ISSUE THREAD

**Issue by kdorheim** [FoRTExperiment/FoRTE-mgmt#68]
[Test disturbance event results](https://github.com/FoRTExperiment/FoRTE-mgmt/files/4245379/ED2.2_Practice_Disturbence_Results.pdf) and the 2020 FoRTE project table. 
![Screen Shot 2020-02-24 at 10 56 04 AM](https://user-images.githubusercontent.com/27299759/75169294-4652da80-56f6-11ea-90ec-419aca241399.png)

This is draft table to show at the meeting in Richmond, my thinking was that light colors mean that the decision has not been made, white means no progress has been made on it and the light green indicates that we are working on it but no decision has been made.

**Comment by kdorheim** [comment]
- [x] prepare slides for the meeting in Richmond (table and disturbance figures)
- [x] fix plots, should sum over the cohort members 
- [ ] Look into the UMBS or GCB results runs 
- [ ] Improve our understanding of how different cohorts are impacted by disturbances. Could we apply disturbances to different cohorts?

**Comment by kdorheim** [comment]
Sum over the Cohort Members 
![image](https://user-images.githubusercontent.com/27299759/75715986-5fcac800-5c9c-11ea-9cd2-58abcaafb276.png)
@bpbond do these units look better?

**Comment by bpbond** [comment]
The LAI is odd, way too low. What's "eaf" I wonder?

The NPP number look sensible. Thanks!

**Comment by kdorheim** [comment]
I think it is m2 leaf / m 2 ground. Should I multiply by the leaf biomass?

**Comment by kdorheim** [comment]
> * Improve our understanding of how different cohorts are impacted by disturbances. Could we apply disturbances to different cohorts?

So looking into `events.f90` there appears to be an option for the forest management called "thin". Upon further investigation this subroutine has not been implemented ☹️ . Things to consider do we want to write this submodule (I'm thinking nahhh) do we want to modify the harvest submodule so that we can apply the harvest to a specific cohort/pft? Do any other process happen on the cohort level that we can look at?

**Comment by bpbond** [comment]
>I think it is m2 leaf / m 2 ground. Should I multiply by the leaf biomass?

Yes, that's the standard units for LAI. But I'd expect a value in the 1-5 range, not 0.01-0.05.

>do we want to write this submodule

Not a chance, I'd say. 😄 

> Do any other process happen on the cohort level that we can look at?

Interesting question and I don't know. This might be something to ping Alexey about.

**Comment by ashiklom** [comment]
I'm still subscribed to these notifications, so I've been lurking 🙈 

For the LAI, what's the variable you're using, and how are you aggregating it? Also, are these runs starting from bare ground or from real initial conditions?

> Do any other process happen on the cohort level that we can look at?

Do you mean in the harvest submodule? If so, no -- my understanding is that submodule uniformly and evenly removes biomass from every cohort. Your best bet for cohort-specific disturbances is to modify initial condition files. In a nutshell, a workflow like:

(1) Run ED2 up to the disturbance.
(2) Read the outputs necessary to create a new initial condition file (DBH, soil moisture, etc.)
(3) Modify these initial condition files to reflect the changes introduced by the disturbance
(4) Restart ED2 from these modified initial condition files

Basically, what I tried to lay out [here](https://github.com/FoRTExperiment/ed4forte/blob/master/documentation/04-disturbance-treatment.md#modifying-initial-condition-files).
